conjure-go-client/httpclient/http2_client_test.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// +build go1.16
```

Comment:
I had to guard this file with the build tag since pre go1.16 the second test case would not succeed, since the fields are not available, and in reality would basically test the same functionality of the first test with wrong assertions.